docs/dev-guide/custom-resource-definitions.md

```diff
@@ -0,0 +1,126 @@
+This page outlines `spec` and `status` fields of all the CRs managed by LVMO
+
+## 1. LVMCluster
+
+- LVMCluster CR represents the volume groups that should be created and managed
+  across nodes with custom node selector, toleration and device selectors
+- Should be created and edited by user in operator installed namespace and only
+  single CR is supported, creation of multiple CRs is erroneous
```

Comment:
```suggestion
- Should be created and edited by user in operator installed namespace.
- only a single instance is supported at present.
```

Reply:
removing this file, same for rest of the comments in this file.